WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorView.swift

```diff
@@ -83,53 +83,14 @@ struct ProductSelectorView: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            HStack {
-                SearchHeader(text: $viewModel.searchTerm, placeholder: Localization.searchPlaceholder, onEditingChanged: { isEditing in
-                    searchHeaderisBeingEdited = isEditing
-                })
-                .padding(.horizontal, insets: safeAreaInsets)
-                .accessibilityIdentifier("product-selector-search-bar")
-                Picker(selection: $viewModel.productSearchFilter, label: EmptyView()) {
-                    ForEach(ProductSearchFilter.allCases, id: \.self) { option in Text(option.title) }
-                }
-                .pickerStyle(.segmented)
-                .padding(.leading)
-                .padding(.trailing)
-                .renderedIf(searchHeaderisBeingEdited)
-                Button(action: {
-                    // TODO: gh-12011
-                }, label: {
-                    Image(uiImage: .scanImage.withRenderingMode(.alwaysTemplate))
-                        .foregroundColor(Color(.brand))
-                })
-                .padding(.trailing)
-                .renderedIf(ServiceLocator.featureFlagService.isFeatureFlagEnabled(.sideBySideViewForOrderForm) &&
-                            presentationStyle == .sideBySide)
-            }
-
-            HStack {
-                Text(viewModel.selectProductsTitle)
-                    .renderedIf(configuration.productHeaderTextEnabled)
-                    .fixedSize()
-                    .padding(.leading)
-                Button(Localization.clearSelection) {
-                    viewModel.clearSelection()
-                }
-                .buttonStyle(LinkButtonStyle())
-                .fixedSize()
-                .disabled(isClearSelectionDisabled)
-                .renderedIf(configuration.multipleSelectionEnabled)
-
-                Spacer()
-
-                Button(viewModel.filterButtonTitle) {
-                    showingFilters.toggle()
-                    ServiceLocator.analytics.track(event: .ProductListFilter.productListViewFilterOptionsTapped(source: source.filterAnalyticsSource))
-                }
-                .buttonStyle(LinkButtonStyle())
-                .fixedSize()
+            if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.sideBySideViewForOrderForm) &&
+                presentationStyle == .sideBySide {
+                productSelectorHeaderTitleRow
+                productSelectorHeaderSearchRow
+            } else {
+                productSelectorHeaderSearchRow
+                productSelectorHeaderTitleRow
```

Comment:
I'm not sure if there's a specific design reason for having these two switched around in phone vs Tablet, but imho it seems like unnecessary complexity and creates an UX discrepancy. We should revisit it with Joe at some point and use the same layout in all devices.

Reply:
I suspect it's because of the title role that `productSelectorHeaderTitleRow` fulfils in side-by-side mode ("Select products" or "2 products selected"), which it doesn't do in the modal presentation. But let's raise it in the design review request.